### ISSUE TITLE
mfa_delete is set incorrectly on aws_s3_bucket_versioning resources

### ISSUE THREAD

**Issue by mogggggg** [minamijoyo/tfedit#40]
When upgrading an `aws_s3_bucket` resource that is using MFA delete the `aws_s3_bucket_versioning` resource that gets created sets the `mfa_delete` value to `true` when it should be `"Enabled"`

**Comment by minamijoyo** [comment]
Hi @mogggggg, Thank you for reporting this!

It seems that the valid value for `mfa_delete` argument was changed from `true` or `false` to `Enabled` or `Disabled`. We need to map it.
I also noticed that the `mfa` argument is required if `mfa_delete` is enabled in v4, but I couldn't find out the `mfa` argument in v3. Do you know where to set `mfa` argument in v3?

https://registry.terraform.io/providers/hashicorp/aws/4.19.0/docs/resources/s3_bucket#mfa_delete
https://registry.terraform.io/providers/hashicorp/aws/4.19.0/docs/resources/s3_bucket_versioning#mfa
> mfa - (Optional, Required if versioning_configuration mfa_delete is enabled) The concatenation of the authentication device's serial number, a space, and the value that is displayed on your authentication device.

**Comment by minamijoyo** [comment]
If I understand correctly, the MFA delete feature requires a root MFA token to call the API.
https://docs.aws.amazon.com/AmazonS3/latest/userguide/MultiFactorAuthenticationDelete.html

I found an old issue when the AWS provider added the `mfa_delete` argument to the `aws_s3_bucket` resource, but apparently, it only manages the state and cannot toggle the feature via Terraform.
https://github.com/hashicorp/terraform-provider-aws/issues/629
https://github.com/hashicorp/terraform-provider-aws/pull/5953

> This cannot be used to toggle this setting but is available to allow managed buckets to reflect the state in AWS

When the `aws_s3_bucket_versioning` resource was added in v4, it also noted that there is no way to enable or disable it.
https://github.com/hashicorp/terraform-provider-aws/pull/5132

Even though the `mfa` argument is now configurable in v4, a token will be expired while it is written to the configuration and then applied.

Based on the above, my understanding is that the `mfa` argument has no meaning and can be ignored for the v4 upgrade tool, and all we need is to map the `mfa_delete` argument.
